src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -380,6 +380,12 @@ func (c controller) postInstallConfigs() error {
 		return errors.Errorf("Timeout while waiting for console to become available")
 	}
 
+	// Aplly post install manifests:
+	err = c.applyPostInstallManifests()
+	if err != nil {
+		c.log.WithError(err).Warnf("Failed to apply post manifests.")
```

Comment:
Shouldn't we return error ?

Reply:
Nope, we should try to create as much manifests as possible and ignore those which failed, but log the error, so user is aware.

Reply:
The user in this case is the user of the Assisted-Installer.
Logging the error will not show this to the user

Reply:
> Nope, we should try to create as much manifests as possible and ignore those which failed, but log the error, so user is aware.

Wouldn't it affect the inspecting of the OLM operators? The service expects the OLM statuses to be either "available" or "failure". In case there will be no status sent - it will wait until timeout, and the user will waste a lot of time

Reply:
That's correct, but in case we return an error here, the assisted-service will report failed installation, and the state of the OLM is not important, if I am not mistaken.

Reply:
event sound nice but i think we already agreed to fail installation with relevant reason